--- linuxconsoletools-1.4.8/utils/inputattach.c.orig	2015-01-18 15:43:41.000000000 +0100
+++ linuxconsoletools-1.4.8/utils/inputattach.c	2015-06-29 21:48:52.931946242 +0200
@@ -622,6 +622,17 @@
 	return 0;
 }
 
+static int wacom_v_init(int fd, unsigned long *id, unsigned long *extra)
+{
+	if (write(fd, "BA19\r", 5) != 5)
+                return -1;
+
+	usleep(100 * 1000);
+
+	setline(fd, CS8, B19200);
+	return 0;
+}
+
 struct input_types {
 	const char *name;
 	const char *name2;
@@ -771,6 +782,9 @@
 { "--wacom_iv",		"-wacom_iv",	"Wacom protocol 4 tablet",
 	B9600, CS8 | CRTSCTS,
 	SERIO_WACOM_IV,		0x00,	0x00,	0,	wacom_iv_init },
+{ "--wacom_v",		"-wacom_v",	"Wacom protocol 5 tablet",
+	B9600, CS8,
+	SERIO_WACOM_V,		0x00,	0x00,	0,	wacom_v_init },
 { NULL, NULL, NULL, 0, 0, 0, 0, 0, 0, NULL }
 };
 
--- linuxconsoletools-1.4.8/utils/serio-ids.h.orig	2015-01-18 15:43:41.000000000 +0100
+++ linuxconsoletools-1.4.8/utils/serio-ids.h	2015-06-29 21:50:29.495275523 +0200
@@ -125,6 +125,9 @@
 #ifndef SERIO_PSMULT
 # define SERIO_PS2MULT		0x3c
 #endif
+#ifndef SERIO_WACOM_V
+# define SERIO_WACOM_V		0x3d
+#endif
 #ifndef SERIO_WACOM_IV
 # define SERIO_WACOM_IV		0x3e
 #endif
--- linuxconsoletools-1.4.8/docs/inputattach.1.orig	2015-01-18 15:43:41.000000000 +0100
+++ linuxconsoletools-1.4.8/docs/inputattach.1	2015-06-29 21:51:16.435273553 +0200
@@ -146,6 +146,9 @@
 .BR \-wacom_iv ", " \-\-wacom_iv
 Wacom protocol IV tablets.
 .TP
+.BR \-wacom_v ", " \-\-wacom_v
+Wacom protocol V tablet (Intuos and Intuos2).
+.TP
 .BR \-war ", " \-\-warrior
 WingMan Warrior.
 .TP
